addons/l10n_ua/data/account_chart_template_config.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="l10n_ua_psbo_chart_template" model="account.chart.template">
+            <field name="currency_id" ref="base.UAH" />
+            <field name="property_account_receivable_id" ref="ua_psbp_361"/>
+            <field name="property_account_payable_id" ref="ua_psbp_631"/>
+            <field name="property_account_expense_categ_id" ref="ua_psbp_901"/>
+            <field name="property_account_income_categ_id" ref="ua_psbp_701"/>
+            <field name="complete_tax_set" eval="True"/>
+            <field name="use_anglo_saxon" eval="True"/>
```

Comment:
I would have suspected continental.  There seems to be a cost of goods sold account...

Reply:
Anglo-Saxon is closer to what we used to have in accounting moves.